molecule/mariadb-version/create.yml

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
create.yml and destroy.yml are distributed with molecule python package since 3.x version, so these files must be removed.

Reply:
Done in all scenarios.